lib/t_web/controllers/device_controller.ex

```diff
@@ -11,6 +11,7 @@ defmodule TWeb.DeviceController do
     :ok =
       Accounts.save_apns_device_id(user_id, user_token, device_token,
         locale: params["locale"],
+        version: params["version"] || "<2.3.0",
```

Comment:
2.3.0 is the version where https://github.com/getsince/test5/commit/6565544afd245a3629e8fe7140739ff81a4916a3 will be released